msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+| `m365_duedatetime` | DateTimeType | Date and time at which the task is due. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z |
+| `m365_hasdescription` | BooleanType | Read-only. Value is true if the details object of the task has a non-empty description and false otherwise. |
+| `m365_id` | StringType | Read-only. ID of the task. It's 28 characters long and case-sensitive. [Format validation](/graph/api/resources/planner-identifiers-disclaimer) is done on the service.|
+| `m365_orderhint` | StringType | Hint used to order items of this type in a list view. The format is defined as outlined in [using order hints in Planner](/graph/api/resources/planner-order-hint-format). |
+| `m365_percentcomplete` | IntegerType | Percentage of task completion. When set to 100, the task is considered completed. |
+| `m365_priority` | IntegerType | Priority of the task. The valid range of values is between 0 and 10, with the increasing value being lower priority (0 has the highest priority and 10 has the lowest priority). Currently, Planner interprets values 0 and 1 as "urgent", 2, 3 and 4 as "important", 5, 6, and 7 as "medium", and 8, 9, and 10 as "low". Additionally, Planner sets the value 1 for "urgent", 3 for "important", 5 for "medium", and 9 for "low". |
+| `m365_planid` | StringType | Plan ID to which the task belongs. |
+| `m365_previewtype` | StringType | This sets the type of preview that shows up on the task. The possible values are: automatic, noPreview, checklist, description, reference. |
+| `m365_referencecount` | IntegerType | Number of external references that exist on the task.|
+| `m365_startdatetime` | DateTimeType | Date and time at which the task starts. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z |
+| `m365_title` | StringType |Title of the task. Primary lookup column |
+
+## Graph Planner Plan
+
+* Entity name: m365_graphplannerplan.
+* Graph resource: [plannerPlan resource type](/graph/api/resources/plannerplan).
+* Sorting isn't supported.
+* Filtering isn't supported.
+* Server driven pagination is supported, with maximum page size being 400.
+
+### Attributes for Graph Planner Plan
+
+| Column  | Dataverse Type | Details |
+|---|---|---|
+| `m365_collaborationrootid` | StringType | Collaboration root ID of the collaboration session the record is associated with. If the record is associated with multiple collaboration sessions this will be returned as a comma delimited string. Note that this attribute won't be returned when retrieving multiple records.|
+| `m365_graphplannerplanid` |StringType ||
```

Comment:
Change to 

| `m365_graphplannerplanId` | guid | Unique identifier of the graph planner plan. |